recipe/meta.yaml

```diff
@@ -154,26 +97,19 @@ requirements:
     - mysql-devel                        # [not win]
     - postgresql                         # [not win]
     - zlib
+    - zstd
+    - zstd-static                        # [not win]
```

Comment:
I think the issues that required this line `zstd-static` have since been resolved by h-ventinari